src/utils.js

```diff
@@ -262,6 +273,111 @@ const recapAlertButton = (court, pacerCaseId, isActive) => {
   return anchor;
 };
 
+// Handles the insertion of the "Create alert" option in the dropdown menu
+const addAlertButtonInRecapAction = (court, pacerCaseId) => {
+  let dropdownHeader = document.getElementById('action-button-dropdown-header')
+  let existingAlertButton = document.getElementById('create-alert-button')
+  if (!existingAlertButton){
+    let alertLi = document.createElement("li");
+    alertLi.setAttribute('id', 'create-alert-button')
+    let createAlert = document.createElement('a')
+    
+    const url = new URL('https://www.courtlistener.com/alert/docket/new/');
+    url.searchParams.append('pacer_case_id', pacerCaseId);
+    url.searchParams.append('court_id', court);
+    
+    createAlert.href = url.toString();
+    createAlert.innerHTML = 'Create alert'
+    createAlert.setAttribute('target', '_blank');
+    alertLi.appendChild(createAlert)
+    dropdownHeader.after(alertLi);
+  }
+};
+
+// Handles the insertion of the "Search this docket" option in the dropdown menu
+const addSearchDocketInRecapAction = (docketNumber)=>{
```

Comment:
I wouldn't call this the `docketNumber`. Docket numbers look like this:

> 12-cv-02239-mjl

Then there are PACER Case ID numbers, which are usually `pacer_case_id` in python, and which look like this:

> 129343909

And finally, there are CourtListener ID numbers, which usually look like `cl_pk` or `cl_id`. Something like:

> 239348934

In this case, it's *not* the docketNumber, it's the CL ID, so we should be very careful about calling it that.

Reply:
Thanks for the clarification! I will change the parameter name to avoid confusion in the future.